### DISCUSSION TITLE
How to connect GlyphVisualizer to Nothing Phone

### DISCUSSION THREAD

**Discussion by InspecteurKepler** [SebiAi/GlyphVisualizer#12] [Q&A] [answered]
Hello,

I'd like to know if it's possible to connect your Nothing Phone directly to GlyphVisualizer, and if so, how?

Thanks a lot

**Comment by SebiAi** [comment, marked as answer]
Hi,

it is currently not possible to do that. The feature is not fully finished yet (mainly "versioning" of the communication protocol still remains) and the Android app still needs an *Glyph Developer Kit* API Key.

Thank you very much for your interest in the feature.
